packages/sdk-js/src/GrowthBook.ts

```diff
@@ -204,12 +246,18 @@ export class GrowthBook<
     this._ctx.attributes = attributes;
     this._render();
     this._updateAllAutoExperiments();
+    if (this._ctx.remoteEval) {
+      this._refresh({}, false, true);
+    }
```

Comment:
What if `setAttributes` is called before loadFeatures? This probably should not call refresh unless features have already been explicitly loaded previously.